rmgpy/tools/fluxdiagram.py

```diff
@@ -60,6 +60,7 @@
 maximumNodePenWidth = 10.0      # The thickness of the border around a node at maximum concentration
 maximumEdgePenWidth = 10.0      # The thickness of the edge at maximum species rate
 radius = 1                      # The graph radius to plot around a central species
+centralReactionCount = None     # The maximum number of reactions to draw from each central species (None draws all)
```

Comment:
Is this the total number of reactions from every species? or just the central species and the remaining species that connect to the central species only get one reaction?

Reply:
Can you change the comment to reflect that?